### PR TITLE
Create SortingSentence.java

### PR DESCRIPTION
**Pull request by adjamalbek**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by adjamalbek** [comment]
all the errors were fixed

File: src/main/java/com/thealgorithms/strings/SortSentenceString.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//                Given a shuffled sentence s containing no more than 9 words, reconstruct and return the original sentence.
+
+package com.thealgorithms.strings;
+import java.util.*;
+
+class SortSentenceString {
+    public String sortSentence(String s) {
+        String answer = "";
+        String[] arr_1 = new StringBuilder(s).reverse().toString().split(" "); Arrays.sort(arr_1);
+        for(String word:arr_1)
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/strings/SortSentenceString.java:17]:
```suggestion
        String[] arr1 = new StringBuilder(s).reverse().toString().split(" ");
        Arrays.sort(arr1);
        for(String word:arr1)
```

**Review by debasishbsws — CHANGES_REQUESTED** [review]

**Comment by adjamalbek** [comment]
Requesting review from @siriak and @yanglbme and @debasishbsws for approval and merging.

**Comment by siriak** [comment]
It's not an algorithm in a sense that there's no problem in sorting something with a built-in function

**Pull request closed without merging**